### DISCUSSION TITLE
option to disable fading hitcircles.

### DISCUSSION THREAD

**Discussion by vetyalmao** [ppy/osu#15123] [Ideas] [answered]
option to disable fading hitcircles.  for not having for people to create the "default-1-10" files.

**Comment by peppy** [comment]
No idea what this means. Please provide more context.

**Reply by bdach** [reply]
I think the OP is referring to the fact that in legacy skinning, people who didn't want the hitcircle expand+fadeout animation w
could instead use a blank hitcircle sprite, and skin the hitcircle numbers to use a sprite with both the actual hitcircle and the number...? And the ask would be to just be able to turn off hitcircle animations without that workaround?

**Reply by vetyalmao** [reply]
> No idea what this means. Please provide more context.

like when you hit a circle it doesnt fade it just disappears. and its a lot of time to do the default-1 to default-10 hitcircles for a non-skinner person.

**Reply by peppy** [reply]
Adjusting animations of gameplay will be part of the skinning customisations, once we get to adding that for gameplay elements in addition to what we have now (HUD only).

**Reply by vetyalmao** [reply]
on osu!lazer or on osu! too?

**Reply by integerrr** [reply]
osu! stable is feature locked, so this will only be on lazer. For stable you can still skin it the old way

**Reply by vetyalmao** [reply]
so only on osu lazer?

**Reply by peppy** [reply]
one and the same my dude

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/5913
